### DISCUSSION TITLE
Kindle PW3 (5.9.7) if I rename "documents" to "books"

### DISCUSSION THREAD

**Discussion by mergen3107** [koreader/koreader#7879] [Q&A] [answered]
Hi! 

I am currently trying to prevent Kindle from adding new pdf and txt files to home screen, which I always read in KOReader. I want to leave the Home screen clean for anything Kindle-proprietary (kxf, axw3, etc) in case I would need them any time in future.

If I rename the "documents" folder to "books" and make this new destination as my HOME in KOReader, will things break in terrible way? Things like History, Reading stats?

Thank you!

**Comment by Frenzie** [comment, marked as answer]
The default home folder is just something that aligns with the defaults on the platform. The program doesn't care which folder you make the default (provided it can be read, etc. ;-) ).

**Reply by mergen3107** [reply]
Thank you! 
So I renamed the folder and replaced all `/mnt/us/documents` with `/mnt/us/books` in `history.lua` and `settings.reader.lua`. Seems to works fine so far! I need to extract cache for thumbnails and metadata again though :D
